source/scene3.cpp

```diff
@@ -55,12 +55,15 @@ void Scene3::render(SDL_Renderer *renderer) {
     }
 #ifdef _XBOX
     case UpdateState::CheckFirmware: {
-      HalReadSMBusValue(I2C_HDMI_ADRESS, I2C_BOOT_MODE, false, &current_boot);
-
-      console->print("Found... Revision: 3C");
-      console->print("Firmware Mode: 0x%02X", current_boot);
+      if(HalReadSMBusValue(I2C_HDMI_ADRESS, I2C_BOOT_MODE, false, &current_boot) != 0) {
```

Comment:
Nit: missing space between `if` and `(`

Reply:
Again, this is replicating other checks that do the exact same thing.